packages/cli/templates/vue2/src/components/FieldPluginExample/index.vue

```diff
@@ -22,7 +43,6 @@
 import ModalToggle from './ModalToggle.vue'
 import Counter from './Counter.vue'
 import AssetSelector from './AssetSelector.vue'
-import Vue from 'vue'
```

Comment:
Same as in React, why was Vue imported here?

Reply:
This might've been unnecessary code within that component file.